scipy/stats/_multivariate.py

```diff
@@ -2810,8 +2741,8 @@ def logpdf(self, x, df, scale):
         """
         dim, df, scale = self._process_parameters(df, scale)
         x = self._process_quantiles(x, dim)
-        _, log_det_scale = self._cholesky_logdet(scale)
-        out = self._logpdf(x, dim, df, scale, log_det_scale)
+        C, log_det_scale = self._cholesky_logdet(scale)
+        out = self._logpdf(x, dim, df, scale, log_det_scale, C)
```

Comment:
The reason to keep the computation of `C` and `log_det_scale` outside of `_logpdf` is so that `invwishart_frozen` can compute `C` and `log_det_scale` upon construction, and then upon `logpdf` call pass them to this `_logpdf`. But it's true that we don't need to pass `scale`, since we don't use it in `_logpdf`.